_about/release-cadence.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Build & Release Cadence
+description: How we manage, number, and support Istio releases.
+weight: 5
+---
+
+We produce new builds of Istio on a daily basis.
+Once a week, one of these daily build undergoes additional
+qualification and gets anointed the weekly release. Once a month, one of the weekly releases undergo
+additional qualifications and gets anointed as the monthly release. Once a quarter or so, one of the monthly
+releases undergoes additional qualifications and gets anointed as a long term supported (LTS) release.
+
+Each release type implies different levels of quality and support:
+
+|Release Type | Support Level | Typical Use
+|-------------+---------------+------------
+|Daily   | No support           | Dangerous, may not be fully reliable. Useful to experiment with.
+|Weekly  | No support           | Dangerous, may not be fully reliable. Useful to experiment with.
+|Monthly | Support is provided for the current monthly release only | Safe to deploy in production. Usually only adopted by bleeding edge users, or users seeking specific features.
+|LTS     | Support is provided until 3 months after the next LTS                | Safe to deploy in production. Users are encouraged to upgrade to these releases as soon as possible.
+
+In this context, *support* means that we will produce patch releases for critical issues and offer technical assistance.
+In addition, longer-term support may be available from 3rd parties and partners.
+
+Prior to Istio 0.8, we increased the product's version number for every monthly release. Effective with 0.8,
+we will increase the product's version number only for LTS releases.
+
+You can find all available releases on our [releases page](https://github.com/istio/istio/releases).
```

Comment:
daily releases can be found in https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/, not on the github release page.

Reply:
or with 1 level of indirection so this page doesn't need to be updated when something changes, I would link to https://github.com/istio/istio/wiki/Daily-builds instead which has additional information